site.ts

```diff
@@ -240,17 +240,17 @@ export function inject({ config, posthog }) {
                     }
                 }
             })
-        }, true)
+        }, forceReload)
     }
 
-    callSurveys(posthog)
+    callSurveys(posthog, true)
 
     let currentUrl = location.href
     if (location.href) {
         setInterval(() => {
             if (location.href !== currentUrl) {
                 currentUrl = location.href
-                callSurveys(posthog)
+                callSurveys(posthog, false)
```

Comment:
I think we need the force true reload actually? Because for the url change example, if it's using the cached surveys then the url survey won't exist even when you go to the correct url page

Reply:
The force reload is for definitions, not for matching though, if I understand correctly.

So it's only when a _new_ survey is added in the UI that you ever really need to reload things.

Reply:
didn't really test this out locally so not sure yet, just felt faster to make a PR than to explain what I meant.

Reply:
you are totally right!